### PR TITLE
MES-4121 - Added Test Report Validation

### PR DESCRIPTION
**Pull request by TomBillingtonUK**
## Description
- Adds Test Report Validation for Cat D and Sub Categories

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/shared/constants/legal-requirements/legal-requirements.constants.ts [hunk 1]
```diff
@@ -10,6 +10,8 @@ export enum legalRequirementsLabels {
   eco = 'Eco (control and planning)',
   vehicleChecks = 'Show me / Tell me',
   uncoupleRecouple = 'Uncouple / Recouple',
+  busStop1 = 'BS (bus stop)',
+  busStop2 = 'BS (bus stop',
```
Comment (szabikr) [inline src/shared/constants/legal-requirements/legal-requirements.constants.ts]: the difference between these two is the missing parenthesise?

**Review by rossfauldsbjss — APPROVED** [review]
looks good

File: src/shared/constants/legal-requirements/legal-requirements.constants.ts (hunk 1 above)
Comment (TomBillingtonUK) [reply src/shared/constants/legal-requirements/legal-requirements.constants.ts]: they should be the same. Nice spot.

File: package.json [hunk 2]
```diff
@@ -214,4 +214,4 @@
       "pre-push": "run-s lint config:dev test gitSecrets scanRepo"
     }
   }
-}
+}
```
Comment (appleabs) [inline package.json:217]: ?

**Review by appleabs — APPROVED** [review]

**Pull request merged**